extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1538,6 +1545,35 @@ export class AmpA4A extends AMP.BaseElement {
    * @param {!Object<string, string|number>=} opt_extraVariables
    */
   emitLifecycleEvent(unusedEventName, opt_extraVariables) {}
+
+  /**
+   * @param {string=} headerValue Method as given in header.
+   */
+  getNonAmpCreativeRenderingMethod(headerValue) {
+    if (headerValue) {
+      return headerValue;
```

Comment:
Let's move the dev warn check here:

if (headerValue && !isEnumValue(XORIGIN_MODE, headerValue)) {
              dev().error('AMP-A4A', `cross-origin render mode header ${headerValue}`);		              
}

Reply:
dev warn or dev error?